scripts/promote.sh

```diff
@@ -11,7 +11,6 @@ packages=(
   @patternfly/react-styles
   @patternfly/react-table
   @patternfly/react-tokens
-  @patternfly/react-topology
```

Comment:
Do we want to leave it in the promotion script so that PF stays responsible for the releases to `@latest` like the other extracted extensions?

Reply:
Yes, thanks